src/ui/manager.tsx

```diff
@@ -26,16 +26,21 @@ const WebReaderContent: React.FC<ReaderReturn & ReaderManagerArguments> = ({
   ...props
 }) => {
   const bgColor = useColorModeValue('ui.white', 'ui.black', 'ui.sepia');
+  const loading = props.isLoading;
+  const startReading = props.state?.readingLocation.start;
+  const endReading = props.state?.readingLocation.end;
```

Comment:
I would rename these to `isAtStart` and `isAtEnd` for readability. They currently seem more like actions than states based on the name